### PR TITLE
MS-1 Configured database connection and liquibase

### PR DESCRIPTION
**Pull request by karencarter29**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -65,6 +79,26 @@
 					</excludes>
 				</configuration>
 			</plugin>
+			<plugin>
+				<!--start with basic information to get Liquibase plugin:
+                include <groupId>, <artifactID>, and <version> elements-->
+				<groupId>org.liquibase</groupId>
+				<artifactId>liquibase-maven-plugin</artifactId>
+				<version>4.20.0</version>
```
Comment (ennnbaev) [inline pom.xml]: Put in property

File: pom.xml [hunk 2]
```diff
@@ -65,6 +79,26 @@
...
+					<!--set values for Liquibase properties and settings
+                    for example, the location of a properties file to use-->
+					<propertyFile>liquibase.properties</propertyFile>
+				</configuration>
+				<dependencies>
+					<!--set up any dependencies for Liquibase to function in your environment for example, a database-specific plugin-->
+					<dependency>
+						<groupId>org.postgresql</groupId>
+						<artifactId>postgresql</artifactId>
+						<version>42.5.4</version>
```
Comment (ennnbaev) [inline pom.xml]: Put in property

File: pom.xml [hunk 3]
```diff
@@ -49,13 +51,25 @@
...
 		</dependency>
+		<dependency>
+			<groupId>org.postgresql</groupId>
+			<artifactId>postgresql</artifactId>
+			<version>42.5.4</version>
+		</dependency>
+		<dependency>
+			<groupId>org.testcontainers</groupId>
+			<artifactId>postgresql</artifactId>
+			<version>1.18.0</version>
```
Comment (ennnbaev) [inline pom.xml]: Put in property

File: pom.xml [hunk 4]
```diff
@@ -49,13 +51,25 @@
 			<artifactId>spring-security-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.postgresql</groupId>
+			<artifactId>postgresql</artifactId>
+			<version>42.5.4</version>
```
Comment (ennnbaev) [inline pom.xml]: Put in property

File: pom.xml [hunk 5]
```diff
@@ -32,6 +33,7 @@
 		<dependency>
 			<groupId>org.liquibase</groupId>
 			<artifactId>liquibase-core</artifactId>
+			<version>4.20.0</version>
```
Comment (ennnbaev) [inline pom.xml]: Put in property

File: src/main/resources/application.properties [hunk 6]
```diff
@@ -1 +1,6 @@
-
+spring.datasource.url=jdbc:postgresql://localhost:5432/${DATABASE_NAME}
```
Comment (S438r) [inline src/main/resources/application.properties:1]: make DATABASE_NAME  = makestart

File: src/main/resources/application.properties [hunk 7]
```diff
@@ -1 +1,6 @@
...
+spring.datasource.username=${DATABASE_USERNAME}
```
Comment (S438r) [inline src/main/resources/application.properties:2]: make DATABASE_USERNAME= makestart

File: src/main/resources/application.properties [hunk 8]
```diff
@@ -1 +1,6 @@
...
+spring.datasource.password=${DATABASE_PASSWORD}
```
Comment (S438r) [inline src/main/resources/application.properties:3]: make DATABASE_PASSWORD= makestart

**Review by ennnbaev — APPROVED** [review]

**Review by KirillShashunkin0999535364 — APPROVED** [review]

**Review by S438r — APPROVED** [review]

**Pull request merged**